src/test/java/core/basesyntax/service/ParserImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.impl.ParserImpl;
+import core.basesyntax.service.impl.ValidatorImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ParserImplTest {
+    private static final Parser PARSER = new ParserImpl(new ValidatorImpl());
+
+    @Test
+    public void parse_Ok() {
+        List<String> data = new ArrayList<>();
+        data.addAll(List.of("type,fruit,quantity", "b,banana,15"));
+        TransactionDto transaction
+                = new TransactionDto("b", new Fruit("banana"), 15);
+        List<TransactionDto> expected = List.of(transaction);
+        List<TransactionDto> actual = PARSER.parse(data);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void parseWithManyLines_Ok() {
```

Comment:
and again

Reply:
I'm not sure that it's strict rule, but I think it's better to change names everywhere. Additional info: https://dzone.com/articles/7-popular-unit-test-naming